resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -47,36 +49,32 @@ private[spark] class Client(
     appArgs: Array[String]) extends Logging {
   import Client._
 
-  private val namespace = sparkConf.get("spark.kubernetes.namespace", "default")
+  private val namespace = sparkConf.get(KUBERNETES_NAMESPACE)
   private val master = resolveK8sMaster(sparkConf.get("spark.master"))
 
   private val launchTime = System.currentTimeMillis
   private val appName = sparkConf.getOption("spark.app.name")
     .orElse(sparkConf.getOption("spark.app.id"))
     .getOrElse("spark")
-  private val kubernetesAppId = s"$appName-$launchTime".toLowerCase.replaceAll("\\.", "-")
-  private val secretName = s"spark-submission-server-secret-$kubernetesAppId"
-  private val secretDirectory = s"$SPARK_SUBMISSION_SECRET_BASE_DIR/$kubernetesAppId"
-  private val sslSecretsDirectory = s"$SPARK_SUBMISSION_SECRET_BASE_DIR/$kubernetesAppId-ssl"
-  private val sslSecretsName = s"spark-submission-server-ssl-$kubernetesAppId"
-  private val driverLauncherSelectorValue = s"driver-launcher-$launchTime"
-  private val driverDockerImage = sparkConf.get(
-    "spark.kubernetes.driver.docker.image", s"spark-driver:$sparkVersion")
-  private val uploadedJars = sparkConf.getOption("spark.kubernetes.driver.uploads.jars")
+  private val kubernetesAppId = sparkConf
+    .get("spark.app.id", s"$appName-$launchTime").toLowerCase.replaceAll("\\.", "-")
```

Comment:
I think if we have something as similarly named as "app id" and "application ID" it should be the same thing everywhere.  Rather than a kubernetes app id and a spark app id being really close in name but slightly different.

Does that mean we need to use the with-timestamp version everywhere?